homeassistant/components/homekit/accessories.py

```diff
@@ -85,34 +97,47 @@ def set_accessory_info(acc, name, model, manufacturer=MANUFACTURER,
 class HomeAccessory(Accessory):
     """Adapter class for Accessory."""
 
-    # pylint: disable=no-member
-
-    def __init__(self, name=ACCESSORY_NAME, model=ACCESSORY_MODEL,
-                 category='OTHER', **kwargs):
+    def __init__(self, hass, name, entity_id, aid, category):
         """Initialize a Accessory object."""
-        super().__init__(name, **kwargs)
-        set_accessory_info(self, name, model)
+        super().__init__(name, aid=aid)
```

Comment:
Are you sure you never need `config` to be passed to the Accessory?

Reply:
Ok, because you used to pass it and stopped with the new code... otherwise all seems similar (hate to ask the obvious, but you did test it? 😄)

Reply:
Test what?
The config was never passed to super().__init__(), since they haven't been keyword arguments before, only normal ones. **kwargs was there to cover any case that I need to pass more arguments from get_accessory to the HAP-python accessory. However only used for aid.